flake.nix

```diff
@@ -64,6 +64,9 @@
         # We include rust-src to ensure rust-analyzer works.
         # See https://discourse.nixos.org/t/rust-src-not-found-and-other-misadventures-of-developing-rust-on-nixos/11570/4
         extensions = [ "rust-src" ];
+        # possible fix for "section too large"
+        # targets = [ "aarch64-apple-darwin" "wasm32-unknown-unknown" ];
+        targets = [ "wasm32-unknown-unknown" ];
```

Comment:
I believe this needs to include all possible targets we support

Reply:
What other targets would that be?